lib/stdlib/src/lists.erl

```diff
@@ -957,6 +957,24 @@ keymap(Fun, Index, [Tup|Tail]) ->
 keymap(Fun, Index, []) when is_integer(Index), Index >= 1, 
                             is_function(Fun, 1) -> [].
 
+-spec enumerate(List1) -> List2 when
+      List1 :: [T],
+      List2 :: [{Index, T}],
+      Index :: integer(),
+      T :: term().
+enumerate(List1) ->
+    enumerate(List1, 0).
```

Comment:
Hmm. Erlang normally does indexing starting from 1, is this really a good default?

Reply:
I was sure that `lists:nth` starts with 0. But you're right, changed to 1